Userland/Libraries/LibLocate/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+set(SOURCES
+    LocateDB.cpp
+    Crawler.cpp
```

Maybe not super important for such small files, but we generally keep `SOURCES` in alphabetical order.

```suggestion
    Crawler.cpp
    LocateDB.cpp
```